app/src/main/res/values/preferences.xml

```diff
@@ -62,6 +62,14 @@
     <string name="preference_title_fast_swiping_enabled">Enable fast swiping</string>
     <string name="preference_summary_fast_swiping_enabled">Allows swiping over multiple rooms.</string>
 
+    <!-- Schedule update notification -->
+    <string name="preference_title_change_statistics_ui_enabled">Enable change statistics dialog</string>
+    <string name="preference_summary_change_statistics_ui_enabled">View change statistics when schedule is updated</string>
```

Comment:
`View change statistics when schedule is updated` -> `Displays the update dialog when the schedule is updated.`